Push server fails to start on Wildfly

The error is:

```
15:08:51,315 ERROR [org.jboss.as.controller.management-operation] (External Management Request Threads -- 1) WFLYCTL0013: Operation ("add") failed - address: ([("deployment" => "powerauth-push-server.war")]) - failure description: {"WFLYCTL0080: Failed services" => {"jboss.deployment.unit.\"powerauth-push-server.war\".undertow-deployment" => "java.lang.RuntimeException: org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'batchSendingConfiguration': Invocation of init method failed; nested exception is java.lang.NoSuchMethodError: com.fasterxml.jackson.databind.ObjectMapper$DefaultTypeResolverBuilder.<init>(Lcom/fasterxml/jackson/databind/ObjectMapper$DefaultTyping;Lcom/fasterxml/jackson/databind/jsontype/PolymorphicTypeValidator;)V
    Caused by: java.lang.RuntimeException: org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'batchSendingConfiguration': Invocation of init method failed; nested exception is java.lang.NoSuchMethodError: com.fasterxml.jackson.databind.ObjectMapper$DefaultTypeResolverBuilder.<init>(Lcom/fasterxml/jackson/databind/ObjectMapper$DefaultTyping;Lcom/fasterxml/jackson/databind/jsontype/PolymorphicTypeValidator;)V
    Caused by: org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'batchSendingConfiguration': Invocation of init method failed; nested exception is java.lang.NoSuchMethodError: com.fasterxml.jackson.databind.ObjectMapper$DefaultTypeResolverBuilder.<init>(Lcom/fasterxml/jackson/databind/ObjectMapper$DefaultTyping;Lcom/fasterxml/jackson/databind/jsontype/PolymorphicTypeValidator;)V
    Caused by: java.lang.NoSuchMethodError: com.fasterxml.jackson.databind.ObjectMapper$DefaultTypeResolverBuilder.<init>(Lcom/fasterxml/jackson/databind/ObjectMapper$DefaultTyping;Lcom/fasterxml/jackson/databind/jsontype/PolymorphicTypeValidator;)V"}}
15:08:51,316 ERROR [org.jboss.as.server] (External Management Request Threads -- 1) WFLYSRV0021: Deploy of deployment "powerauth-push-server.war" was rolled back with the following failure message:
{"WFLYCTL0080: Failed services" => {"jboss.deployment.unit.\"powerauth-push-server.war\".undertow-deployment" => "java.lang.RuntimeException: org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'batchSendingConfiguration': Invocation of init method failed; nested exception is java.lang.NoSuchMethodError: com.fasterxml.jackson.databind.ObjectMapper$DefaultTypeResolverBuilder.<init>(Lcom/fasterxml/jackson/databind/ObjectMapper$DefaultTyping;Lcom/fasterxml/jackson/databind/jsontype/PolymorphicTypeValidator;)V
    Caused by: java.lang.RuntimeException: org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'batchSendingConfiguration': Invocation of init method failed; nested exception is java.lang.NoSuchMethodError: com.fasterxml.jackson.databind.ObjectMapper$DefaultTypeResolverBuilder.<init>(Lcom/fasterxml/jackson/databind/ObjectMapper$DefaultTyping;Lcom/fasterxml/jackson/databind/jsontype/PolymorphicTypeValidator;)V
    Caused by: org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'batchSendingConfiguration': Invocation of init method failed; nested exception is java.lang.NoSuchMethodError: com.fasterxml.jackson.databind.ObjectMapper$DefaultTypeResolverBuilder.<init>(Lcom/fasterxml/jackson/databind/ObjectMapper$DefaultTyping;Lcom/fasterxml/jackson/databind/jsontype/PolymorphicTypeValidator;)V
    Caused by: java.lang.NoSuchMethodError: com.fasterxml.jackson.databind.ObjectMapper$DefaultTypeResolverBuilder.<init>(Lcom/fasterxml/jackson/databind/ObjectMapper$DefaultTyping;Lcom/fasterxml/jackson/databind/jsontype/PolymorphicTypeValidator;)V"}}
```

--- a/powerauth-push-server/src/main/webapp/WEB-INF/jboss-deployment-structure.xml
+++ b/powerauth-push-server/src/main/webapp/WEB-INF/jboss-deployment-structure.xml
@@ -4,6 +4,7 @@
 		<exclude-subsystems>
 			<!-- disable the logging subsystem because the application manages its own logging independently -->
 			<subsystem name="logging" />
+			<subsystem name="jaxrs" />
 		</exclude-subsystems>
 
 		<dependencies>

